logprep/processor/list_comparison/rule.py

```diff
@@ -78,6 +79,16 @@ def _get_list_search_base_path(self, list_search_base_path):
     def init_list_comparison(self, list_search_base_path: Optional[str] = None):
         """init method for list_comparision lists"""
         list_search_base_path = self._get_list_search_base_path(list_search_base_path)
+        if list_search_base_path.startswith("http"):
```

Comment:
As this behaviour seems to always apply when a base path starting with http is used (and in order to make this accessible), could you please also add the template functionality to the processor documentation?

Reply:
DONE